src/Uno.Templates/content/unoapp/Directory.Build.props

```diff
@@ -52,6 +52,17 @@
     <AndroidMaterialVersion  Condition=" '$(AndroidMaterialVersion)' == '' ">1.9.0.2</AndroidMaterialVersion>
     <AndroidXNavigationVersion  Condition=" '$(AndroidXNavigationVersion)' == '' ">2.6.0.1</AndroidXNavigationVersion>
     <!--#endif-->
+    <!--#if (analyzers)-->
+
+    <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+    <EnableNETAnalyzers>true</EnableNETAnalyzers>
+    <AnalysisMode>Recommended</AnalysisMode>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
We probably should remove this, and instead pass it here:

https://github.com/unoplatform/uno.templates/blob/c098cae34ce35b9025f7762eaa81691b5f291f93/build/templates/package-validation.yml#L79